hi-ml/testhiml/testhiml/test_model_trainer.py

```diff
@@ -87,6 +88,33 @@ def _get_trainer_arguments() -> Dict[str, Any]:
     assert isinstance(storing_logger, StoringLogger)
 
 
+@pytest.mark.fast
+def test_create_lightning_trainer_limit_batches() -> None:
```

Comment:
Would there be a way to test that the trainer has recognised the parameter we passed in, e.g. check the number of train/val batches we get? Or is this not necessary because we trust PL?

Reply:
I would not test at that level. As a default, I feel we should trust that PL does what it says on the box.

Reply:
I have added some very basic checks that make sure that the PL Trainer objects have the expected (limited) number of batches